src/RA_httpthread.cpp

```diff
@@ -366,33 +367,34 @@ BOOL RAWeb::DoBlockingHttpGet(const std::string& sRequestedPage, std::string& Re
                 nullptr,
                 WINHTTP_NO_REFERER,
                 WINHTTP_DEFAULT_ACCEPT_TYPES,
-                0);
+                0_dw);
 
             // Send a Request.
             if (hRequest != nullptr)
             {
                 BOOL bResults = WinHttpSendRequest(hRequest,
                     L"Content-Type: application/x-www-form-urlencoded",
-                    0,
+                    0_dw,
                     WINHTTP_NO_REQUEST_DATA, //WINHTTP_NO_REQUEST_DATA,
-                    0,
-                    0,
-                    0);
+                    0_dw,
+                    0_dw,
+                    0_dw);
```

Comment:
I'm still not sold on this. No matter how many times I see "0_dw", it just looks ugly. I'm going to take a page from Google's C++ guidelines and ask that you remove these.

https://google.github.io/styleguide/cppguide.html#Operator_Overloading

> Overload operators judiciously. Do not create user-defined literals.

> User-defined literals allow the creation of new syntactic forms that are unfamiliar even to experienced C++ programmers.

Reply:
Sure, they're gone.